cpp/fury/util/logging.h

```diff
@@ -35,7 +35,7 @@ enum class FuryLogLevel {
   DEBUG = -1,
   INFO = 0,
   WARNING = 1,
-  ERROR = 2,
+  ERR = 2,
```

Comment:
Could you attach the error you face of the original `ERROR` identifier?

Reply:
Hmm it seems here: https://github.com/apache/fury/actions/runs/11643078405/job/32423399925#step:5:1894

Reply:
Just took a glance, and I think that maybe the Cython generated code defines a macro named `ERROR` and it's included before this header so that weird thing happens. Not familiar with related code here, maybe @penguin-wwy has ideas about this.